### PR TITLE
Add "widescreen" page layout

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

**Comment by lucperkins** [comment]
@yurishkuro You can see an example of this layout here: http://deploy-preview-43--jaegertracing.netlify.com/docs/client-libraries/

As you can see, I also split up the features matrix into separate tables, just as an experiment. Let me know how you feel about that.

**Comment by yurishkuro** [comment]
> As you can see, I also split up the features matrix into separate tables, just as an experiment. Let me know how you feel about that.

just ran it by the team, people seem to like the original version better due to strict column alignment.

**Comment by lucperkins** [comment]
@yurishkuro So you want to keep the widescreen layout but go back to the original table format?

**Comment by yurishkuro** [comment]
yes, and also move the whole features section to a separate `/client-features` page (here the hierarchical left side menu would be handy, but for now we can simply not include the matrix in the menu).

File: themes/jaeger-docs/layouts/shortcodes/featuresTable.html [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
-        {{- else }}
-        {{- $supported := default "`?`" $supported }}
-        <td>{{ $supported | markdownify }}</td>
-        {{- end }}
-        {{- end }}
-      </tr>
+  <tbody>
+    {{- range $featureGroups }}
+    <tr>
+      <th class="feature-table-header">{{ .description | markdownify }}</th>
```
Comment (yurishkuro) [inline themes/jaeger-docs/layouts/shortcodes/featuresTable.html]: missed `colspan`

**Comment by yurishkuro** [comment]
I moved the feature matrix to a separate page, so this is almost ready to merge, the only issue is that the new page shows at the end of the menu. I removed the `rank` field but it still shows up - @lucperkins

**Comment by lucperkins** [comment]
@yurishkuro That should fix the sorting issue. Apparently Hugo has a built-in `ByWeight` sorting mechanism that gets us exactly what we want.

**Comment by yurishkuro** [comment]
thanks, this will do for now. I booked #45 for the hierarchical menu.

**Pull request merged**